include/revng/ABI/Definition.h

```diff
@@ -147,6 +147,13 @@ type: struct
       Setting this to true disables usage of vector registers entirely.
     type: bool
 
+  - name: AllowPuttingSimilarlyAlignedStackArgumentsTogether
```

Comment:
1. This field is use before being introduced. Does this pass `check-branch`?
2. What does "similarly" mean?

Reply:
1. Ah, thanks for noticing, something slipped through then. My bad.

2. Yeah, on the fresh look, the name is awful, what do you think about something like `StackArgumentsUseNormalStructAlignmentRules`?

Reply:
Works, but I'd use "Regular" instead of "Normal".

Reply:
Done.